reliability-engineering/terraform/modules/concourse-secrets-admin/variables.tf

```diff
@@ -0,0 +1,25 @@
+data "aws_caller_identity" "account" {}
+
+variable "deployment" {
+  type = string
+}
+
+variable "name" {
+  type = string
+}
+
+variable "kms_key_arn" {
+  type = string
+}
+
+variable "connector_host_path" {
+  type = string
+}
+
+variable "client_id" {
+  type = string
+}
+
+variable "team_id" {
```

Comment:
Suggest calling this `trusted_github_team_id`, and making it an integer (to dissuade people from trying to stick `re-autom8` in it and wonder why it's not working)

Reply:
Lord knows at some point there will be a team wanting to grant it to multiple github teams and we'll have to worry about that but we'll cross that bridge when we come to it...

Reply:
accepted and embodied.